vonk/features/preload.rst

```diff
@@ -7,7 +7,7 @@
 Preloading a set of resources
 =============================
 
-.. caution:: Preload in Vonk 3.0.0-beta1 is only available for STU3. Expect it to be enabled for R4 in the next beta.
+.. caution:: Preload in Vonk 3.0.0 is only available for STU3. Expect it to be enabled for R4 in the next version.
```

Comment:
Come to think of it: I'm not convinced we should keep Preload. Let's not promise it for R4.